### ISSUE TITLE
New Feature Pitches (Happy to collaborate/contribute)

### ISSUE THREAD

**Issue by bloniaszp** [craddm/eegUtils#136]
Hi there,

I would love the chance to contribute several features to eegUtils. I have the following suggestions:

1) Multitaper Spectral Analysis to give people parametric control over their spectral estimates 
2) Functional Connectivity capability: Coherency (complex valued), Coherence (modulus of coherency), and Global Coherence (frequency domain analog to PCA)
3) Add a maximum likelihood based ICA (e.g., picard), in addition to FAST or infomax.

Otherwise, happy to make these as their own stand alone package if you feel their outside of your original scope.

Best regards,
Patrick

**Comment by craddm** [comment]
Hi Patrick,

These are great ideas for features! Out of the three, I think particularly multitapers and additional ICA methods might fit nicely in the existing scope. I don't have a lot of time to work on the package these days (I'm not in the field any more) so won't get round to doing them myself, but those are things I'd have been interested in adding. Connectivity not so much, but only because I've never used it so am not familiar with it. Can totally see it as being within scope. 

I was starting to come round to the idea of adding more features as external packages though, as the package was getting a little unwieldy - I had thought about refactoring it to a suite of packages rather than one monolithic package. But not likely that will happen now, either.

An external package for maximum likelihood based ICA might be good, as it feels like the kind of thing that might have applications beyond EEG data and that people might want to use without needing the rest of the package.

So, TL;DR - could fit either way, as standalone packages that we can interface with eegUtils, or as extensions to the existing codebase. 

Cheers,
Matt
